cypress.config.ts

```diff
@@ -29,8 +27,8 @@ export default defineConfig({
             return config;
         },
         baseUrl: "http://localhost:3200",
-        video: false,
-        screenshotOnRunFailure: false,
+        video: true,
+        screenshotOnRunFailure: true,
```

Comment:
We will now start outputting videos and screenshots of tests so it's easier to debug